mautic/mautic
# Removing Mautic\CoreBundle\Controller\FormController

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:

This PR is removing an "abstract" class `FormController` that several controllers were still extending. I refactored those to use the replacement. This class was deprecated in Mautic 2.3.

#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Test asset download is working.
3. Test that the campaign builder works, you can add, edit, remove source and events.
4. Test email opens and redirects
5. Test form builder can add/edit/remove actions and fields
6. Forms can be submitted
7. Form results can be viewed, sorted
8. Landing pages can be viewed by contacts
9. Point triggers can be created/updated/deleted
10. Webhooks can be created/updated/deleted
